src/ts/components/seekbarcontroller.ts

```diff
@@ -0,0 +1,111 @@
+import { UIUtils } from '../uiutils';
+import { DOM } from '../dom';
+import { PlayerAPI } from 'bitmovin-player';
+import { VolumeController } from '../volumecontroller';
+
+export enum SeekBarType {
+    Vod,
+    Live,
+    Volume,
+}
+
+const changeRangeValue = (
+    value: number,
+    range: { min: number, max: number },
```

Comment:
It maybe would make sense to extract `{ min: number, max: number }` into a `Range` interface.